lib/src/Pos/Launcher/Resource.hs

```diff
@@ -241,11 +242,11 @@ getRealLoggerConfig LoggingParams{..} = do
         Nothing -> identity
         Just _  -> \lc -> lc { _lcBasePath = lpHandlerPrefix }
     overrideConsoleLog :: LoggerConfig -> LoggerConfig
-    overrideConsoleLog = case lpConsoleLog of
-        Nothing    -> identity
-        Just True  -> (<>) (defaultInteractiveConfiguration Info)
+    overrideConsoleLog src = case lpConsoleLog of
+        Nothing    -> src
+        Just True  -> src <> (defaultInteractiveConfiguration Info)
                       -- add output to the console with severity filter >= Info
-        Just False -> identity
+        Just False -> src <> disabledConsoleConfiguration
```

Comment:
`disabledConsoleConfiguration` should be a filter on the incoming LoggerConfig and remove all backends that write to the console.